# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../apmd/hotfixes.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./apmsleep.c.orig	2001-05-11 22:30:08.000000000 +0200
+++ ./apmsleep.c	2002-09-15 17:38:51.000000000 +0200
@@ -42,11 +42,15 @@
  *
  */
 
+typedef unsigned char u8;
+typedef unsigned short u16;
+typedef unsigned int u32;
+
 #include <stdio.h>
 #include <linux/version.h>
 
 #if LINUX_VERSION_CODE > KERNEL_VERSION(2,2,0)
-#include <asm/spinlock.h>
+// #include <asm/spinlock.h>
 #endif
 
 #include <linux/mc146818rtc.h>
